src/scripts/train_generative_model.py

```diff
@@ -6,9 +6,11 @@
 import pandas as pd
 from omegaconf import DictConfig
 
-from european_values.data_loading import load_evs_wvs_data
+from european_values.data_loading import load_evs_trend_data, load_evs_wvs_data
 from european_values.data_processing import process_data
-from european_values.generative_training import train_generative_model
+from european_values.generative_training import (
+    train_generative_model,  # <-- This was missing!
```

Comment:
No internal comments in the code please 🙂 

```suggestion
    train_generative_model
```